### PR TITLE
Define version in GitHub release action

### PR DESCRIPTION
**Pull request by nipafx**
WIP

### CODE REVIEW

**Comment by aepfli** [comment]
```
 > Task :githubRelease FAILED
:githubRelease (Thread[Execution worker for ':',5,main]) completed. Took 0.014 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Some problems were found with the configuration of task ':githubRelease' (type 'GithubReleaseTask').
  - Type 'GithubReleaseTask' property 'githubToken' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'githubToken'.
      2. Mark property 'githubToken' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'GithubReleaseTask' property 'writeToken' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'writeToken'.
      2. Mark property 'writeToken' as optional.
    
    Please refer to https://docs.gradle.org/7.0/userguide/validation_problems.html#value_not_set for more details about this problem.

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output. Run with --scan to get full insights.
```

there are still some token definitions missing :(

File: .github/workflows/release-build.yml [hunk 1]
```diff
@@ -27,14 +31,17 @@ jobs:
...
           ORG_GRADLE_PROJECT_signingKey: ${{ secrets.JUNIT_PIONEER_PGP_SECRET_KEY }}
           ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.JUNIT_PIONEER_PGP_PASSWORD }}
           # used by `io.github.gradle-nexus.publish-plugin` to publish to Maven Central
           ORG_GRADLE_PROJECT_sonatypeUsername: ${{ secrets.NEXUS_TOKEN_USERNAME }}
           ORG_GRADLE_PROJECT_sonatypePassword: ${{ secrets.NEXUS_TOKEN_PASSWORD }}
+          ORG_GRADLE_PROJECT_version: ${{ github.event.inputs.version }}
         with:
-          arguments: publishToSonatype closeAndReleaseSonatypeStagingRepository githubRelease
+          arguments: publishToSonatype closeSonatypeStagingRepository githubRelease -i
+          # arguments: publishToSonatype closeAndReleaseSonatypeStagingRepository githubRelease
```
Comment (aepfli) [inline .github/workflows/release-build.yml:44]:
```suggestion
          arguments: publishToSonatype closeAndReleaseSonatypeStagingRepository githubRelease
```

**Comment by nipafx** [comment]
[This release build](https://github.com/junit-pioneer/junit-pioneer/actions/runs/790107444) worked, so it looks like we can merge this.

**Comment by nipafx** [comment]
FYI: This can't be merged until documentation is updated.

**Pull request merged**